examples/fizzbuzz/main.c

```diff
@@ -0,0 +1,206 @@
+#include <qmlbind.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+/*!
+ * \file main.c
+ * \brief This example shows how to wrap fizzbuzz into a metaclass with `libqmlbind`.
+ */
+
+#define UNUSED(x) (void)(x)
+
+#define ENOUGH 10 // FIXME: use correct value
+
+
+typedef struct fizzbuzz
+{
+    int input;
+    char *result;
+    qmlbind_signal_emitter *signalEmitter;
+} fizzbuzz_t;
+
+
+void on_changed(fizzbuzz_t *fb, const qmlbind_value *input)
+{
+    int i = (int) qmlbind_value_get_number(input);
```

Comment:
Before getting the number, we should get the string representation and see whether it's an empty string. I think it's very irritating that deleting the current number, which means entering "", is interpreted as 0 and therefore as fizzbuzz, and then displayed as "last fizzbuzz is: " because it uses the value of the text field.

(Yes, this also applies to ruby-qml's version).